tests/shared/file-extensions.spec.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2016 resin.io
```

Comment:
God, I'll never get a chance to do that. Mind sending a massive search and replace PR for me? :P